apps/ws/src/socket/usecases/external-services-route/external-services-route.usecase.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable } from '@nestjs/common';
+
+import { MessageRepository } from '@novu/dal';
+import { ChannelTypeEnum } from '@novu/shared';
+import { BullMqService } from '@novu/application-generic';
+
+import { ExternalServicesRouteCommand } from './external-services-route.command';
+import { WSGateway } from '../../ws.gateway';
+
+@Injectable()
+export class ExternalServicesRoute {
```

Comment:
can we cover it with tests?

Reply:
it should be covered in our e2e but will look into creating e2e for this usecase  🙃